.github/actions/run-tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15-alpine
+FROM golang:1.16-alpine
```

Comment:
Is this needed here?

Reply:
Yes, because my IDE suggestion made me use `os.WriteFile` which did not exist prior to Go 1.16 and [functions as a replacement for `ioutil.WriteFile`](https://golang.org/pkg/io/ioutil/#WriteFile).

Reply:
Your IDE thought it was looking at Go 1.16 and suggested inappropriate replacements; so you thought rather than try and fix the IDE's idea of the code, make the code conform to it ...

Reply:
Using Go 1.16 is unlikely to break anything, but perhaps explain why that change happens here, in the commit message.

Reply:
> Your IDE thought it was looking at Go 1.16 and suggested inappropriate replacements; so you thought rather than try and fix the IDE's idea of the code, make the code conform to it ...

In another revision of this code I was using a version of [`filepath#WalkDir`](https://golang.org/pkg/path/filepath/#WalkDir) that is also only available since Go 1.16 so while taking that and

> Using Go 1.16 is unlikely to break anything

in mind, I went ahead and made the change. I will update the commit message to better reflect my structural thinking.